### DISCUSSION TITLE
Org info

### DISCUSSION THREAD

**Discussion by T1xx1** [abbrcode/abbreviations-in-code#90] [General]
I'm doing the .github org profile. Which information should I include?

- Name (Abbrcode)
- Description (?)
- Discussions link and abbrincode link
- Owner (@kisvegabor)
- Members (@philipeachille, @T1xx1, @oood)

And...?

**Comment by oood** [comment]
Maybe an introduction?

**Comment by kisvegabor** [comment]
Please add yourselves as owners :slightly_smiling_face:

**Reply by T1xx1** [reply]
We didn't have the idea so technically we're not. I can put us as mantainers.

**Reply by kisvegabor** [reply]
I really won't have much time to be active here in the near future. So I'd be happy to give more rights and "titles" to you.

**Reply by T1xx1** [reply]
If you want.

@oood do you agree?

**Reply by oood** [reply]
I don't have much time these weeks either, and BTW, I'd rather be called a contributor.

**Reply by oood** [reply]
But recently, I don't deserve the title of contributor with almost zero contribution, so I am willing to accept that I don't have any title for the time being.
